modules/core/02-client/client/cli/cli.go

```diff
@@ -48,7 +48,8 @@ func NewTxCmd() *cobra.Command {
 		NewUpdateClientCmd(),
 		NewSubmitMisbehaviourCmd(), // Deprecated
 		NewUpgradeClientCmd(),
-		NewCmdSubmitRecoverClientProposal(),
+		newCmdSubmitRecoverClientProposal(),
+		newCmdScheduleIBCUpgradeProposal(),
```

Comment:
should func names have `Cmd` suffix, rather than `NewCmd`.

something like:

```suggestion
		newSubmitRecoverClientProposalCmd(),
		newScheduleIBCUpgradeProposalCmd(),
```

Reply:
sdk seems slightly mixed (looking at gov and bank), suffix does read better imo too